packages/amplify-provider-awscloudformation/lib/push-resources.js

```diff
@@ -41,7 +41,7 @@ async function run(context, resourceDefinition) {
 
     validateCfnTemplates(context, resources);
 
-    await packageResources(context, resources);
+    await packageResources(context, resources, projectDetails);
```

Comment:
What is projectDetails?

Reply:
I had it for a previous implementation that is now refactored, so it's not needed anymore. This will be updated on the next push!